#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-check-_REENTRANT.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Rather than #defining _REENTRANT if it's not yet defined, #error out
## DP: See Debian bug #380739.

@DPATCH@
--- ACE_wrappers.orig/ace/config-lite.h 	2006-09-10 21:51:38.312980000 +0200
+++ ACE_wrappers/ace/config-lite.h 	2006-09-10 21:36:50.861517750 +0200
@@ -68,7 +68,7 @@
 # if defined (ACE_MT_SAFE) && (ACE_MT_SAFE != 0)
 #   define ACE_MT(X) X
 #   if !defined (_REENTRANT)
-#     define _REENTRANT
+#     error _REENTRANT has to be defined!
 #   endif /* _REENTRANT */
 # else
 #   define ACE_MT(X)
